### ISSUE TITLE
[TBC] Psychic scream / Demonic Sacrifice

### ISSUE THREAD

**Issue by BestServedMoist** [cmangos/issues#1087]
**Spells**: 
Psychic Scream - ID: 10890
Demonic Sacrifice - ID: 18788 

**Current behavior**: 
Removes aura [Demonic Sacrifice] on warlock, when the player gets feared by [Psychic Scream].

**Expected behavior**: 
Aura should not be removed

**Steps to reproduce**:
-> Make a priest and a warlock class. 

-> Learn the IDs listed above and duel the other class, the warlock should sacrifice his current demon pet for the buff. 

-> Now make the priest cast Psychic Scream, this should remove the Sacrifice buff on the warlock.

Tested with different pets. This is a recurring bug.

**TL;DR**:
Psychic Scream overriding aura Demonic Sacrifice

**Operating system**: 
Ubuntu LTS 16.04

**Note**:
@Warlockbugs might be working on this already.
[//]: # (If this is a crash report, include the crashlog from a debug build with https://gist.github.com/)

**Comment by Warlockbugs** [comment]
Fixed in upcoming aura rewrite. Hang on in there.

**Comment by Warlockbugs** [comment]
Annoying age-old "same icon" stuff got rid of in https://github.com/cmangos/mangos-tbc/commit/f14a867aa666df8869f9cf8d37ae654389d35ba9.
